File: Java/practice2/src/Test/DatabaseConnectionTest.java
package Test;

import DataSources.Data.DatabaseConnection;
import io.github.cdimascio.dotenv.Dotenv;
import org.junit.Before;
import org.junit.Test;

import java.sql.Connection;

public class DatabaseConnectionTest {

    private Dotenv vault;

    @Before
    public void setup(){
        vault = Dotenv.load();
    }
    @Test
    public void ShouldReturnDatabaseConnection(){
        assert DatabaseConnection.connect(true);
        assert DatabaseConnection.connect(false);
    }
}

File: Kotlin/practice2/src/Test/DatabaseConnectionTest.kt
package Test

import DataSources.Data.DatabaseConnection
import io.github.cdimascio.dotenv.Dotenv
import io.github.cdimascio.dotenv.dotenv
import junit.framework.TestCase.assertEquals
import org.junit.Before
import org.junit.Test
import org.junit.runner.RunWith
import org.junit.runners.JUnit4
import java.sql.Connection

@RunWith(JUnit4::class)
class DatabaseConnectionTest {
        private lateinit var vault: Dotenv
    @Before
    fun setup(){
        this.vault = dotenv()
    }

    @Test
    fun ShouldConnectAndChangeItStatusToTrue(){
        val dbStatus: Boolean = DatabaseConnection.connectionStatus


        assertEquals(true, dbStatus)
    }
}